### PR TITLE
[[ Documentation ]] Tweak all docs for standalone inclusion APIs

### PR DESCRIPTION
**Pull request by livecodeali**
- Remove all notes which state that script library inclusions are implemented as buttons in a hidden group (bug 17637)
- Add associations to all library API entries
- Update stock text referring to Standalone Settings GUI:

```
>*Important:*  The <**name**> <**type**> is part of the 
> <**library_name** library>. To ensure that the <**type**> works in a 
> <standalone application>, you must include this 
> <LiveCode custom library|custom library> when you create your 
> <standalone application|standalone>. In the Inclusions pane of the 
> <Standalone Application Settings> window, make sure the "**library_name**" 
> library checkbox is checked.
```
- Add a (basic) glossary entry for SSL & Encryption library
- Change LiveCode Custom library type to glossary and update references

### CODE REVIEW

**Comment by montegoulding** [comment]
@livecodeali I see that you have ensured the entire reference `<...>` is all on one line. Is that required? My docs wrapper script will need some changes if so.

**Comment by livecodeali** [comment]
@montegoulding No, all the munging is done before the links are resolved, so it's fine to wrap within `<...>` - I tend to do it when I'm manually editing just for vague readability purposes I suppose.

File: docs/dictionary/command/revXMLAddNode.lcdoc [hunk 1]
```diff
@@ -32,8 +34,14 @@ The result: If the <revXMLAddNode> <command> encounters an error, the <result> i
 Description:
 Use the <revXMLAddNode> <command> to add a <node> to an <XML tree>.
 
->*Important:*  The <revXMLAddNode> <command> is part of the <XML library>. To ensure that the <command> works in a <standalone application>, you must include this <LiveCode custom library|custom library> when you create your <standalone application|standalone>. In the Inclusions section on the General screen of the <Standalone Application Settings> window, make sure "XML Library" is selected in the list of script libraries.
+>*Important:*  The <revXMLAddNode> <command> is part of the 
+> <Video library>. To ensure that the <command> works in a
```
Comment (peter-b) [inline docs/dictionary/command/revXMLAddNode.lcdoc]: I think you meant `<XML library>` here.

Comment (peter-b) [inline docs/dictionary/command/revXMLAddNode.lcdoc]: Several of the XML dictionary entries exhibit this problem.

**Comment by livecodeali** [comment]
@peter-b Thanks, fixed!

**Comment by peter-b** [comment]
@livecode-vulcan review ok 46b2155

@livecodeali If I remember correctly you already updated the guides...

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 46b21557aedddf2b4afcaba718077a1a5e4a549b

**Comment by livecode-vulcan** [comment]
😎 test success 46b21557aedddf2b4afcaba718077a1a5e4a549b
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2598)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2608)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2593)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2609)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2615)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2621)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2633)

**Comment by livecodeali** [comment]
@peter-b You do remember correctly!

**Pull request merged**